Exec/RegTests/Spray-Conv/multiRuns.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+
+# Usage:
+#   ./multiRuns.py --test_dir DummyTest --input_file Spray-Conv.inp --pele_exec PeleC-Spray-Conv --run_cmd "mpiexec -np 4"
+
+# Input:
+#   * input_file: name of the input file, assumes it ends in "inp" if not given
+#   * test_dir: name of test directory, test directory must contain the executable, input file, and gridfiles
+#   * run_cmd: command to run test cases with, like 'mpiexec -np 4' or just './'
+#   * pele_exec: PeleC executable, assumed to be in test_dir
+#   * max_error: Maximum allowable error when using FCompare
+#   * fcomp_exec: FCompare executable
+
+# "Internal" user input
+#   * bsizes: box sizes to test
+#   * pltfiles: names of plot files to do compare on
+#   * gridfiles: names of gridfiles used
+
+
+import sys
+import os
+import shutil
+import argparse
+import numpy as np
```

Comment:
doens't look like numpy is used here, can you remove?

Reply:
Yep!